README.md

```diff
@@ -78,7 +78,9 @@ Add the following to your `.bashrc` (or `.profile` on Mac):
 
 ```
 function _update_ps1() {
-    PS1="$(~/go/bin/powerline-go -error $?)"
+    if [ -e ~/go/bin/powerline-go ]; then
```

Comment:
If you want to be more general, you could use `$GOPATH/bin/powerline-go`

Reply:
That seems reasonable, though since it's a static binary (at least for me) then maybe it's simpler to not depend upon a variable?

In any case, apply the second commit too, if that's preferable :+1: